schema@v1.3.0/schema.json

```diff
@@ -99,6 +99,15 @@
         }
       }
     },
+    "crs": {
+      "description": "Coordinate reference system",
+      "type": "string",
+      "enum": [
+        "EPSG:28992",
+        "EPSG:4326",
+        "EPSG:7415"
```

Comment:
nog handig erbij (staan ook standaard in onze WFS:)

* `EPSG:3857` (Spherical Mercator (Google Maps)
* `EPSG:4258` (ETRS89, European Terrestrial Reference System 1989)

Reply:
Zonder enum kan elke willekeurige string. dat lijkt me niet zo handig dan breekt de toolchain als mensen een typo maken. Als we alle epsgs willen gaan toestaan misschien even meenemen in de bespreking over de 3d feature. Wel vet als het gewoon kan

Reply:
ok, goede, duidelijk! Misschien de waarden in de validator meenemen, en niet perse in de schema definitie. Maar dat is voor later :-)